Meta: Add preload/OWNERS

snuggs opened the pull request:
AKA the #ironic PR.

![capture d ecran 2017-11-26 a 01 23 53](https://user-images.githubusercontent.com/38223/33237704-8d19ebd2-d248-11e7-82b9-1e4f30d65203.png)

- Add missing `preload/OWNERS`
- Request to be notified of `preload/**` changes.

Following instructions within http://web-platform-tests.org/introduction.html#github

<!-- Reviewable:start -->

<!-- Reviewable:end -->

@sideshowbarker @igrigorik @zcorpan @foolip Feel free to remove yourself if you would prefer to not have the extra spam. However, interest in `rel=preload` is increasing and `wpt-pr-bot` doesn't lie :-)

snuggs committed:
Meta: Add preload/OWNERS

`preload/OWNERS`

  - [x] Add missing `preload/OWNERS`
  - [x] Add @sideshowbarker to preload/OWNERS
  - [x] Add @igrigorik to preload/OWNERS
  - [x] Add @zcorpan to preload/OWNERS
  - [x] Add @foolip to preload/OWNERS
  - [x] Add @snuggs to preload/OWNERS

  References:
    http://web-platform-tests.org/introduction.html#github

w3c-bots commented:
# Build PASSED

Started: 2017-11-10 03:41:35
Finished: 2017-11-10 03:44:24


<h2>Failing Jobs</h2>
<ul>

<li>chrome:unstable</li>

<li>MicrosoftEdge:14.14393</li>

</ul>




View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/20458)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/299987721)

snuggs commented:
😳  Pardon the rebase error which triggered review requests from CSS team.

foolip commented:
I haven't done any work on this spec, so I'm opting out :)

(Too many reviewers seems to make it less likely that anyone will react, too.)

snuggs committed:
Meta: Remove @foolip from preload/OWNERS

snuggs commented:
Aggreed @foolip and no harm no foul. Tough skin in OSS is a requirement. You are free to go. lolz. https://github.com/w3c/web-platform-tests/pull/8142/commits/a668f78e8e33294671d46f99ddbac0ddf84c1927

foolip commented:
Maybe @yoavweiss would also makes sense, as he's listed as spec editor? @zcorpan @sideshowbarker, have you done any work around this spec?

snuggs committed:
Meta: Add @yoavweiss to preload/OWNERS

  Addresses:
    - https://github.com/w3c/web-platform-tests/pull/8142#issuecomment-347005427

snuggs commented:
@foolip Fixed in https://github.com/w3c/web-platform-tests/pull/8142#issuecomment-347005427

yoavweiss commented:
Happy to review preload tests! :)

snuggs commented:
@yoavweiss @foolip I think we just need one more for now... Or @yoavweiss and I can just rock, paper, scissors if there is a stalemate on a decision... :mountain: :spiral_notepad: ✂️

snuggs committed:
Meta: Update preload/Owners with confirmed maintainers

foolip commented:
Now merged!
